CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 
 ## DFX
 
+### fix(frontend-canister): Allow overwriting default HTTP Headers for assets in frontend canister 
+
+Allows to overwrite `Content-Type`, `Content-Encoding`, and `Cache-Control` HTTP headers with custom values via `.ic-assets.json5` config file. Example `.ic-assets.json5` file:
+```json5
+[
+    {
+        "match": "web-gz.data.gz",
+        "headers": {
+            "Content-Type": "application/octet-stream",
+            "Content-Encoding": "gzip"
+        }
+    }
+]
+```
+
+
```

Comment:
The changelog should include a note about upgrading the asset canister (meaning they will go through the upgrade process), including the new module hash.